git_pile/genbranch.py

```diff
@@ -38,7 +38,12 @@ def genbranch(config, args):
 
 
 def genbranch_with_exit_stack(config, args, exit_stack):
-    if not config.check_is_valid():
+    if args.no_config:
```

Comment:
in the previous commit maybe add genbranch to the list of subcommands that can be called without a config?  That way we keep track of subcommands that are supposed to handle that.